examples/text_classification/train.py

```diff
@@ -154,9 +160,17 @@ def test(data_):
                           embed_dim, len(train_dataset.get_labels())).to(device)
     criterion = torch.nn.CrossEntropyLoss().to(device)
 
-    train(lr, num_epochs, train_dataset)
-    test(test_dataset)
+    # split train_dataset into train and valid
+    train_len = int(len(train_dataset) * split_ratio)
+    sub_train_, sub_valid_ = \
```

Comment:
I'm not sure this works if split_ratio is 1 - have you tried running it with '1'?